onert-micro/luci-interpreter/include/luci_interpreter/InterpreterConfigure.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ONERT_MICRO_INTERPRETER_CONFIGURE_H
+#define ONERT_MICRO_INTERPRETER_CONFIGURE_H
+
+namespace luci_interpreter
+{
+
+class InterpreterConfigure
+{
+public:
+  void setAllocateInputValue(bool allocate_input) { _allocate_input = allocate_input; }
+  bool getAllocateInputValue() const { return _allocate_input; }
+
+  void useStaticMemoryManager(uint32_t input_buf_size, uint32_t temp_buf_size,
+                              uint32_t output_buf_size)
+  {
+    _use_static_manager = true;
+    _input_buf_size = input_buf_size;
+    _temp_buf_size = temp_buf_size;
+    _output_buf_size = output_buf_size;
+  }
```

Comment:
Since you plan to read buffer size values from circle file, and remove the fields, what do you think about interface like this:
```suggestion
  InterpreterConfigure &useStaticMemoryManager()
  {
    _use_static_manager = true;
    return *this;
  }
  
  InterpreterConfigure &useDynamicMemoryManager()
  {
    _use_static_manager = false;
    return *this;
  }

  InterpreterConfigure &configStaticMemoryManager(uint32_t input_buf_size, uint32_t temp_buf_size,
                              uint32_t output_buf_size)
  {
    assert(_use_static_manager);
    _input_buf_size = input_buf_size;
    _temp_buf_size = temp_buf_size;
    _output_buf_size = output_buf_size;
    return *this;
  }
```

Reply:
I prefer the second variants, thank you! :)